src/stepper/cn2.f90

```diff
@@ -55,8 +61,24 @@ subroutine cn2_set_diffusion(self, dt, vorch)
              endif
 
             !$omp parallel workshare
-            diss = one / (one + dfac * hdis)
+            vdiss = one / (one + dfac * vhdis)
```

Comment:
This does not look like the usual exponential integrating factor (see my code, evolution.f90).  Is this ok?

Reply:
That is another integrator. We provide two time integrators.